dev-client/src/components/home/SiteMap.tsx

```diff
@@ -42,9 +42,7 @@ type SiteCalloutProps = {
 const SiteCallout = ({site, closeCallout}: SiteCalloutProps) => {
   const {t} = useTranslation();
   const project = useSelector(state =>
-    site.projectId === undefined
-      ? undefined
-      : state.project.sites[site.projectId],
+    site.projectId === undefined ? undefined : state.site.sites[site.projectId],
```

Comment:
nit: same as prev comment
```suggestion
    site.projectId === undefined ? undefined : state.project.projects[site.projectId],
```